capella2polarion/converters/converter_config.py

```diff
@@ -70,14 +72,24 @@ def __init__(self):
         self.diagram_config: CapellaTypeConfig | None = None
         self.__global_config = CapellaTypeConfig()
 
-    def read_config_file(self, synchronize_config: t.TextIO):
+        self._type_prefix: str
+        self._role_prefix: str
+
+    def read_config_file(
+        self,
+        synchronize_config: t.TextIO,
+        type_prefix: str = "",
+        role_prefix: str = "",
+    ):
         """Read a given yaml file as config."""
+        self._type_prefix = type_prefix
+        self._role_prefix = role_prefix
```

Comment:
we should not have `_type_prefix` and `_role_prefix` as members of this class. We just need to pass the `role_prefix` to `_force_link_config` and the `type_prefix` to `set_diagram_config` and we should be all good